README.md

```diff
@@ -18,6 +18,40 @@ Usage example:
 # ... and open a browser on http://127.0.0.1:5000
 ```
 
+## Installation
+
+### From Python `pypi` packages
+
+The easiest, stable way to install Speculos is with `pip`:
```

Comment:
should we add here that the command needs to be run in speculos repo (root) ?

Reply:
For source install, sure.